NextcloudTalk/SettingsTableViewController.swift

```diff
@@ -247,11 +248,12 @@ class SettingsTableViewController: UITableViewController, UITextFieldDelegate {
     // MARK: Notifications
 
     @objc func appStateHasChanged(notification: NSNotification) {
-        let appState = notification.userInfo?["appState"]
-        if let appState = appState as? AppState {
-            self.adaptInterfaceForAppState(appState: appState)
-        }
-    }
+             let appState = notification.userInfo?["appState"]
+             if let rawAppState = appState as? UInt32 {
+                 let appState = AppState(rawValue: rawAppState)
+                 self.adaptInterfaceForAppState(appState: appState)
+             }
+         }
```

Comment:
Please remove this change or indent it correctly, as this is already merged to master.